sal-common/src/main/java/org/ow2/proactive/sal/model/AbstractNode.java

```diff
@@ -20,13 +20,19 @@
 @MappedSuperclass
 public abstract class AbstractNode implements Node {
 
+    // JSON field constants
+    public static final String JSON_ID = "id";
+
+    public static final String JSON_NODE_CANDIDATE = "nodeCandidate";
+
     @Id
     @GeneratedValue(generator = "system-uuid")
     @GenericGenerator(name = "system-uuid", strategy = "uuid")
     @Column(name = "ID")
-    @JsonProperty("id")
+    @JsonProperty(JSON_ID)
     protected String id = null;
 
     @OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.REFRESH)
+    @JsonProperty(JSON_NODE_CANDIDATE)
```

Comment:
Just `@JsonProperty` is enough as by default it maps with the field name.
Or you could use `@JsonProperty(required = true)` if the field shouldn't be null.

+ remove the static Strings if not used anymore.